test/mobile/features/ios/ios_notify.feature

```diff
@@ -0,0 +1,172 @@
+
+        When I tap the "Notify caught exception" button
+        Then I wait to receive an error
+
+        # Exception details
+        And the error payload field "events" is an array with 1 elements
+        And the exception "errorClass" equals "IndexOutOfRangeException"
+        And the exception "message" equals "Index was outside the bounds of the array."
+        And the event "unhandled" is false
+        And the event "severity" equals "warning"
+        And the event "severityReason.type" equals "handledException"
+
+        # Stacktrace validation
+        And the error payload field "events.0.exceptions.0.stacktrace" is a non-empty array
+        And the event "exceptions.0.stacktrace.0.method" ends with "ReporterBehavior.NotifyCaughtException()"
+        And the event "exceptions.0.stacktrace.0.lineNumber" equals 0
+        And the error payload field "events.0.threads" is null
+
+        # App data
+        And the event "app.id" equals "com.bugsnag.unity.mazerunner"
+        And the event "app.releaseStage" equals "production"
+        And the event "app.type" equals "iOS"
+        And the event "app.version" equals "1.2.3"
+#        And the event "app.versionCode" equals "1"
+#        And the error payload field "events.0.app.duration" is not null
+        And the error payload field "events.0.app.durationInForeground" is not null
+#        And the event "app.inForeground" equals "true"
+#        And the event "app.isLaunching" equals "false"
+#        And the event "app.memoryUsage" is not null
+#        And the event "app.name" equals "maze_runner"
+#        And the event "app.lowMemory" equals "false"
+
+        # Device data
+        And the event "device.jailbroken" is false
+        And the event "device.id" is not null
+        And the error payload field "events.0.device.id" is stored as the value "device_id"
+        And the event "device.locale" is not null
+        And the event "device.manufacturer" is not null
+        And the event "device.model" is not null
+        And the event "device.osName" equals "iOS"
+        And the event "device.osVersion" is not null
+        And the event "device.runtimeVersions" is not null
+
+#        And the event "device.totalMemory" is not null
+        And the event "device.freeDisk" is not null
+        And the event "device.freeMemory" is not null
+#        And the event "device.orientation" equals "portrait"
+        And the event "device.time" is not null
+#        And the event "device.locationStatus" is not null
+#        And the event "device.emulator" equals "false"
+#        And the event "device.networkAccess" is not null
+#        And the event "device.charging" is not null
+#        And the event "device.screenDensity" is not null
+#        And the event "device.dpi" is not null
+#        And the event "device.screenResolution" is not null
+#        And the event "device.brand" is not null
+#        And the event "device.batteryLevel" is not null
```

Comment:
Some of these (including the other comment blocks) look like fields which are only ever set on Android - seems worthwhile to remove them instead of leaving them commented out in the iOS tests.